tools/pre-commit

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
added to the [wiki](https://github.com/unibas-dmi-hpc/SPH-EXA/wiki/Contributing-to-SPH-EXA#git-hooks-and-clang-format)